assets/timber.scss.liquid

```diff
@@ -244,6 +244,14 @@ $socialIconFontStack: 'icons';
 }
 
 /*============================================================================
+  Layer promotion mixin for creating smoother animations with higher FPS.
+==============================================================================*/
+@mixin promote-layer($properties: transform) {
+  -webkit-transform: translateZ(0); // translateZ hack
```

Comment:
Any way/need to add an IE solution to this? I've never really thought about it until now